### DISCUSSION TITLE
Mountain Duck Won't Mount / (forward slash)anymore.

### DISCUSSION THREAD

**Discussion by EdwardRitzema** [iterate-ch/cyberduck#14798] [Q&A]
Mountain Duck Won't Mount:  /(forward slash) anymore after version 4.12.3 (Mac OS) So I Can't Update

**Comment by tatze96** [comment]
Does the issue persist within the current release version (4.14.1) of Mountain Duck?

If so, please enable [debug logging](https://docs.cyberduck.io/mountainduck/support/#debug-log), recreate the described behavior, and send us any available *mountainduck.log* and *mountainduck-\*.log.zip* files for further investigation to support@mountainduck.io. Ensure to include the ticket reference GH-14798.

You can ZIP the log files or share them through a cloud storage of your choice if they are too big for sending.

Additionally, add the following information:
- A screenshot of the error message if there are any present.
- The protocol (e.g. WebDAV, S3, OneDrive, etc.) in use.
